linux/memory_hot_add_basic/memory_hot_add_basic.yml

```diff
@@ -31,9 +31,9 @@
           vars:
             skip_msg: "Skip testcase '{{ ansible_play_name }}', memory hotadd is not supported on 32bit Debian"
```

Comment:
Replace `32bit Debian` in `skip_msg` with `{{ guest_os_ansible_distribution }} {{ guest_os_ansible_architecture }}`.